recipes/libglvnd/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+from conans import Meson, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import functools
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+class LibGlvndConan(ConanFile):
+    name = "libglvnd"
+    description = "The GL Vendor-Neutral Dispatch library"
+    license = "LicenseRef-LICENSE"
```

Comment:
https://github.com/NVIDIA/libglvnd#libglvnd

```suggestion
    license = "MIT"
```